airflow/www/templates/airflow/dags.html

```diff
@@ -76,9 +76,21 @@ <h2>DAGs</h2>
             <th width="12">
               <span id="pause_header" class="material-icons text-muted" title="Use this toggle to pause a DAG. The scheduler won't schedule new tasks instances for a paused DAG. Tasks already running at pause time won't be affected.">info</span>
             </th>
-            <th>DAG</th>
-            <th>Schedule</th>
-            <th>Owner</th>
+            <th>
+              <a href="{{ url_for('Airflow.index', status=request.args.get('status', 'all'), search=request.args.get('search', None), tags=request.args.get('tags', None), sortBy='dag_id', orderBy='asc' if request.args.get('sortBy') != 'dag_id' or request.args.get('orderBy', 'desc') != 'asc' else 'desc' ) }}">
+                DAG
```

Comment:
It would probably be good to add icons to these links to give them a bit more visual distinction.

```suggestion
                DAG<span class="material-icons">unfold_more</span>
```

![image](https://user-images.githubusercontent.com/3267/96515020-8db44b00-1232-11eb-92dc-94889f7572c1.png)

Reply:
Added; will add screenshots to the PR body.